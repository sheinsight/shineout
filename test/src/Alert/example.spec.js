import exampleTest from '../../example'

describe('Alert[snapshot]', () => {
  exampleTest('Alert')
})
