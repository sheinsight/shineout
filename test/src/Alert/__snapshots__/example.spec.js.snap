// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert[snapshot] Snapshot test: Alert[site/pages/components/Alert/example-1-base.tsx] 1`] = `
<div>
  <div>
    <h3>
      Title
    </h3>
    Some content.
  </div>
</div>
`;

exports[`Alert[snapshot] Snapshot test: Alert[site/pages/components/Alert/example-2-type.tsx] 1`] = `
<div>
  <div>
    <div>
      Success Type.
    </div>
  </div>
  <div>
    <div>
      Info Type.
    </div>
  </div>
  <div>
    <div>
      Warning Type.
    </div>
  </div>
  <div>
    <div>
      Danger Type.
    </div>
  </div>
</div>
`;

exports[`Alert[snapshot] Snapshot test: Alert[site/pages/components/Alert/example-3-close.tsx] 1`] = `
<div>
  <div>
    <a>
      <svg>
        <path />
      </svg>
    </a>
    <div>
      Alert onClose=true
    </div>
  </div>
  <div>
    <a>
      <svg>
        <path />
      </svg>
    </a>
    <div>
      Alert onClose=function
    </div>
  </div>
  
</div>
`;

exports[`Alert[snapshot] Snapshot test: Alert[site/pages/components/Alert/example-4-icon.tsx] 1`] = `
<div>
  <div>
    <div>
      <svg>
        <path />
      </svg>
    </div>
    <div>
      Success Type.
    </div>
  </div>
  <div>
    <div>
      <svg>
        <path />
      </svg>
    </div>
    <div>
      Info Type.
    </div>
  </div>
  <div>
    <div>
      <svg>
        <path />
      </svg>
    </div>
    <div>
      Warning Type.
    </div>
  </div>
  <div>
    <div>
      <svg>
        <path />
      </svg>
    </div>
    <div>
      Danger Type.
    </div>
  </div>
  <div>
    <div>
      <svg>
        <path />
      </svg>
    </div>
    <div>
      <h3>
        Set iconSize
      </h3>
      iconSize=24
    </div>
  </div>
</div>
`;
